client/App.jsx

```diff
@@ -1,21 +1,48 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import axios from 'axios';
+import styled from 'styled-components';
+import TopDetails from './TopDetails.jsx';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
+    this.className = props.className;
     this.state = {
+      phone: {
+        name: 'default phone name',
+        productCode: 'product code',
+        capacities: [{ size: 'size1' }, { size: 'size2' }],
+        carriers: [{ name: 'Verizon' }, { name: 'Sprint' }],
+        colors: [{ name: 'Blue' }, { name: 'Red' }]
+      }
+    };
+  }
 
-    }
+  componentDidMount() {
+    axios.get('http://localhost:3002/phones/9')
```

Comment:
A couple of notes here. 

First: you'll probably find it preferable to work with relative addresses for requests. You may run into some of the reasons once you hit deployment or proxying, but for a quick one check out what happens when you try to load your app locally at 127.0.0.1:3002 instead of localhost:3002.

Second: Keep in mind that your default state values are what will be displayed to the user in the event of a failed API call. Is this what you want them to see in that case?

Third: I'm a little confused by the forced /9 endpoint. Is this a temporary thing?